manifests/pipecd/grafana-dashboards/piped/overview.json

```diff
@@ -153,10 +160,119 @@
       ],
       "title": "Running versions",
       "type": "piechart"
+    },
+    {
+      "datasource": null,
+      "description": "List all activated pipeds",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "custom": {
+            "align": "auto",
+            "displayMode": "auto"
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "version"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "go_version"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Value"
+            },
+            "properties": [
+              {
+                "id": "displayName",
+                "value": "count"
+              }
+            ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "project"
+            },
+            "properties": [
+              {
+                "id": "mappings",
+                "value": [
+                  {
+                    "options": {
+                      "match": "empty",
+                      "result": {
+                        "index": 0,
+                        "text": "-"
+                      }
+                    },
+                    "type": "special"
+                  }
+                ]
+              }
+            ]
```

Comment:
nit: I would prefer to change the header "piped" to "id" but it's on you

![image](https://user-images.githubusercontent.com/19730728/132648012-6e73f036-f9bb-4269-afde-975d231fd828.png)

Reply:
Sure, lets me adopt it. 👍

Reply:
addressed by https://github.com/pipe-cd/pipe/pull/2449/commits/27c044dc4faadff898a974b3d53424b9eaa62d0a 🙏